api.md

```diff
@@ -840,7 +846,150 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+> - [CREATE VIEW](#cagg_create_view)
+> - [ALTER VIEW](#cagg_alter_view)
+> - [REFRESH](#cagg_refresh_view)
+> - [DROP VIEW](#cagg_drop_view)
+
+## CREATE VIEW [](cagg_create_view) 
+`CREATE VIEW` statement is used to create continuous aggregates.
+
+The syntax is:
+``` sql
+    CREATE VIEW <view_name> [ (column_name [, ...] ) ] 
+    WITH ( timescaledb.continuous [, timescaledb.<option> = <value>])
+    AS
+    <select-query>
+    
+    <select_query> is of the form :
+    SELECT <grouping-cols>, <aggregate-functions>
+    FROM <hypertable>
+    [WHERE ... ]
+    GROUP BY <time_bucket( <const-value>, partition-col-of-hypertable>),
+             [ optional grouping exprs>
+    [HAVING ...]
+```
+
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+| `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
+ | `WITH` clause | This clause specifies [options](#create-view-with) for the continuous aggregate view.|
+ | `<select_query>`| A `SELECT` query that uses the specified syntax. |
+
+#### `WITH` clause options [](create-view-with)
+|Name|Description|Type|Default|
+|---|---|---|---|
+|**Required**||
```

Comment:
These need additional `|` so that the tables complete.

![Screen Shot 2019-05-02 at 5 32 33 PM](https://user-images.githubusercontent.com/12288273/57108334-609f7000-6d00-11e9-8aec-9f8e1ba60be2.png)

Reply:
fixed.